app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  # logs in the user
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by(id: user_id)
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by(id: user_id)
+      if user&.authenticated?(cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
+    end
+  end
+
+  # Returns true if the the user is logged in, false if the user in logged out
+  def logged_in?
+    !current_user.nil?
+  end
+
```

Comment:
The `current user` in a SessionHelper that is included in the Application Controller. This is ok, however against the project specification instruction. kindly add those to the application controller

Reply:
Oay @imhta